README.md

```diff
@@ -195,24 +200,144 @@ Key       | Type    | Default     | Description
 
 Key       | Type     | Default     | Description
 :--       | :--      | :--         | :--
-`in`      | Array    | `undefined` | An array of strings to valid paths for CSS files. All files will remain untouched. A new atomized string is produced for `out`/`result`.
+`in`      | Array    | `undefined` | An array of strings to valid paths for CSS files. All files will remain untouched. A new atomized string is produced for `out`/hooks.
+`data`    | String   | `undefined` | A string of CSS to be atomized. Files are provived via `in` are concatenated with `data` at the end, then atomized and sent to `out`/hooks.
 `out`     | String   | `undefined` | A string file path output. If file exists it will be overwritten with the atomized styles from `in` and/or `data`
-`data`    | String   | `undefined` | A string of CSS to be atomized. Atomized styles are passed to `out` and `result`.
-`result`  | Function | `undefined` | A function, if supplied, will be called with the combined atomized results from `data` and the files form `in` as the first argument, and an error for the second argument. You can create a promise in this function to be returned if you like.
 
 **Markup Task API:**
+
 Key       | Type     | Default     | Description
 :--       | :--      | :--         | :--
-`in`      | String   | `undefined` | Path to an HTML file.
+`in`      | String   | `undefined` | Path to an HTML file to be processed.
+`data`    | String   | `undefined` | A string of markup to be processed. This is appended to the end of the `in` file contents if both are provided.
 `out`     | String   | `undefined` | Path where the modified version of the `in` file will be stored. If file already exists, it will be overwritten.
```

Comment:
```suggestion
`out`     | String   | `undefined` | Path where the modified version of the `in` file will be stored. If file already exists, it will be overwritten.
`hooks`   | Object   | `{}`        | Lifecycle callback hooks (documented in next section)
```